### PR TITLE
Only build most used binaries with make build

### PR DESCRIPTION
**Pull request by ivange94**
# Description of change

Closes https://github.com/iotaledger/wasp/issues/1194

### CODE REVIEW

**Review by dessaya — APPROVED** [review]
Nice! Do the same with `make install`? :pray:

File: .gitignore [hunk 1]
```diff
@@ -26,3 +26,5 @@ goshimmer.log
 
 *.trie
 *.snapshot
+snapshot
+gascalibration
```
Comment (lmoe) [inline .gitignore:30]: Do we still have this gascalibration binary? I've thought it's inside the wasp-cli now.

**Review by lmoe — APPROVED** [review]
Neat!

File: .gitignore (hunk 1 above)
Comment (ivange94) [reply .gitignore:30]: I'm not aware of `gascalibration` being made part of `wasp-cli`. Do you have a link to the commit?

Comment (lmoe) [reply .gitignore:30]: Nope, seems like I was mistaken. :)

**Review by dessaya — APPROVED** [review]

**Pull request merged**